core/deployment/src/test/java/io/quarkiverse/operatorsdk/test/HelmChartGeneratorTest.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkiverse.operatorsdk.test;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkiverse.operatorsdk.test.sources.SimpleCR;
+import io.quarkiverse.operatorsdk.test.sources.SimpleReconciler;
+import io.quarkiverse.operatorsdk.test.sources.SimpleSpec;
+import io.quarkiverse.operatorsdk.test.sources.SimpleStatus;
+import io.quarkus.test.QuarkusProdModeTest;
+
+public class HelmChartGeneratorTest {
+
+    @RegisterExtension
+    static final QuarkusProdModeTest config = new QuarkusProdModeTest()
+            .setApplicationName("helm-chart-test")
+            .withApplicationRoot(
+                    (jar) -> jar.addClasses(SimpleReconciler.class, SimpleCR.class, SimpleSpec.class, SimpleStatus.class));
+
+    @Test
+    void generatesHelmChart() {
```

Comment:
I think it would be good to assert that the resources are indeed generated.

Reply:
yes, will take a look on this, there was an issue with the test, and though an e2e will be enough, bit will give it an other try. Thx!

Reply:
added simple asserts, not sure if we should check the templates in depth in this case.